src/View/SSTemplateParser.php

```diff
@@ -528,7 +528,7 @@ function match_CallArguments ($stack = array()) {
     			$_64 = NULL;
     			do {
     				if (( $subres = $this->whitespace(  ) ) !== FALSE) { $result["text"] .= $subres; }
-    				if (substr($this->string ?? '',$this->pos ?? 0,1) == ',') {
+    				if (substr($this->string,$this->pos,1) == ',') {
```

Comment:
Removing a bunch of the null coalescing operators added by the PHP 8.1 code writing - can we guarantee that these will always not be null?  It's going to be safer to keep these in

We can run this file through https://github.com/emteknetnz/php81-fixer to put these back in

Reply:
Those must not have been added to the .peg file then. Those changes will have to be made on the .peg file, and then the .php file will need to be recompiled.

Reply:
I've created a PR to `4` to updated the .peg file.  We should merge this new PR, then merge it up to `5`, then rebase the PR on `5`

Reply:
I don't think the order really matters but I'm happy to do it that way.